sections/main-product.liquid

```diff
@@ -382,101 +382,7 @@
-                            {{ 'products.product.add_to_cart' | t }}
-                          {%- else -%}
-                            {{ 'products.product.sold_out' | t }}
-                          {%- endif -%}
-                        </span>
-                        <div class="loading-overlay__spinner hidden">
-                          <svg
-                            aria-hidden="true"
-                            focusable="false"
-                            class="spinner"
-                            viewBox="0 0 66 66"
-                            xmlns="http://www.w3.org/2000/svg"
-                          >
-                            <circle class="path" fill="none" stroke-width="6" cx="33" cy="33" r="30"></circle>
-                          </svg>
-                        </div>
-                      </button>
-                      {%- if block.settings.show_dynamic_checkout -%}
-                        {{ form | payment_button }}
-                      {%- endif -%}
-                    </div>
-                  {%- endform -%}
-                </product-form>
-
-                {{ 'component-pickup-availability.css' | asset_url | stylesheet_tag }}
-
-                {%- assign pick_up_availabilities = product.selected_or_first_available_variant.store_availabilities
-                  | where: 'pick_up_enabled', true
-                -%}
-
-                <pickup-availability
-                  class="product__pickup-availabilities no-js-hidden quick-add-hidden"
-                  {% if product.selected_or_first_available_variant.available and pick_up_availabilities.size > 0 %}
-                    available
-                  {% endif %}
-                  data-root-url="{{ routes.root_url }}"
-                  data-variant-id="{{ product.selected_or_first_available_variant.id }}"
-                  data-has-only-default-variant="{{ product.has_only_default_variant }}"
-                >
-                  <template>
-                    <pickup-availability-preview class="pickup-availability-preview">
-                      {% render 'icon-unavailable' %}
-                      <div class="pickup-availability-info">
-                        <p class="caption-large">{{ 'products.product.pickup_availability.unavailable' | t }}</p>
-                        <button class="pickup-availability-button link link--text underlined-link">
-                          {{ 'products.product.pickup_availability.refresh' | t }}
-                        </button>
-                      </div>
-                    </pickup-availability-preview>
-                  </template>
-                </pickup-availability>
-              </div>
-
-              <script src="{{ 'pickup-availability.js' | asset_url }}" defer="defer"></script>
+              {%- render 'buy-buttons', block: block, product: product, product_form_id: product_form_id, show_pickup_availability: true -%}
```

Comment:
It's not working for me on the `Puppy` product. I haven't seen why so far but trying to check. Ill update the comment when I do 👍

Reply:
This is the error by the way. I've seen it this week for another reason somewhere 🤔 But can't remember where. 
<details>
<summary>Screenshot</summary>

![](https://screenshot.click/24-54-n95bn-zqnwb.png)
</details>

Reply:
Oh I remember why 😅 It's not related to your PR but the varian option name `format` which cause issue and was brought up in [another issue](https://github.com/Shopify/dawn/issues/2138) 👍